src/DetailsView/reports/components/instance-list-group-header.tsx

```diff
@@ -43,11 +48,15 @@ export const InstanceListGroupHeader = NamedSFC<InstanceListGroupHeaderProps>('I
     };
 
     const renderDescription = () => {
-        return <span className="rule-details-description">{props.ruleResult.description}</span>;
+        return (
+            <span className="rule-details-description" id={ariaDescribedBy}>
+                {props.ruleResult.description}
+            </span>
+        );
     };
 
     return (
-        <div role="heading">
+        <div role="heading" aria-level={4}>
```

Comment:
I'm guessing this line will have conflicts.
On my PR I'm passing the aria level value. I guess you can have a default value = 4 and override it if the `ariaLabel` property is not null.

Reply:
updated